doc/CHANGES.md

```diff
@@ -3,6 +3,8 @@
 ## Highlighted Features
 
 * Add missing erb tags (Sam Smith)
+* Link to the #internal_review section of the `help/unhappy page` instead of
```

Comment:
backtick ends in the wrong place, and you missed your name off the log message :)